### PR TITLE
update web-components to use variables from css-library

### PR DESCRIPTION
**Pull request by it-harrison**
## Chromatic

https://2236-remove-variables-file--60f9b557105290003b387cd5.chromatic.com

---

## Description
This PR makes changes to allow for the removal of the [variables.scss file](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/_variables.scss) from the web-component package, which is a duplicate of [this file](https://github.com/department-of-veterans-affairs/component-library/blob/bef641ef2c85077d4a013d5fe32030fd176912bc/packages/css-library/dist/tokens/scss/variables.scss#L5) in css-library.
A [companion PR](https://github.com/department-of-veterans-affairs/component-library/pull/946) for the css-library package _**must be merged and published before**_ this PR is merged.

Closes [2236](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2236)

## Testing done
local testing done by using verdaccio to publish css-library and then consume in web-components:
**css-library**:

Screenshot 2023-11-13 at 2 00 13 PM

**verdaccio**:
Screenshot 2023-11-13 at 1 28 03 PM

**web-components - build**:

Screenshot 2023-11-13 at 1 48 29 PM

**screenshots of some components that use variables from css-library** 

`va-statement-of-truth`
Screenshot 2023-11-13 at 2 12 12 PM

`va-accordion`
Screenshot 2023-11-13 at 2 10 30 PM

`va-button`
Screenshot 2023-11-13 at 2 14 07 PM

`<va-segmented-progress-bar`

Screenshot 2023-11-13 at 2 20 51 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "serve": "stencil build --dev --watch --serve"
   },
   "dependencies": {
+    "@department-of-veterans-affairs/css-library": "workspace:^",
```
Comment (it-harrison) [inline packages/web-components/package.json:30]: update the version here to be "1.0.0" when `css-library` published

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by micahchiang — COMMENTED** [review]
@it-harrison - Thanks for working on this! Could you provide a little more context for how these changes were tested? Could you please also attach some screenshots to indicate visual parity was maintained when changing the variable import? Thank you!

**Review by jamigibbs — COMMENTED** [review]
Is it possible to clean up any lone-variables hanging around individual components like this now? 

https://github.com/department-of-veterans-affairs/component-library/blob/5aefd6dcbff87864fae2ef21af98a18cc6acef90/packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss#L4

**Comment by jamigibbs** [comment]
Do we want this to be added to the [component-library release notes](https://github.com/department-of-veterans-affairs/component-library/releases)? If not, then the PR should be tagged `ignore-for-release`.

**Comment by it-harrison** [comment]
> @it-harrison - Thanks for working on this! Could you provide a little more context for how these changes were tested? Could you please also attach some screenshots to indicate visual parity was maintained when changing the variable import? Thank you!

@micahchiang ticket has been updated to show process

> @it-harrison - Thanks for working on this! Could you provide a little more context for how these changes were tested? Could you please also attach some screenshots to indicate visual parity was maintained when changing the variable import? Thank you!

@micahchiang ticket has been updated with testing information / screenshots

**Comment by jamigibbs** [comment]
@it-harrison I just wanted to make sure you saw my comment above. Is there a path forward yet for dealing with any existing variables in individual components? https://github.com/department-of-veterans-affairs/component-library/pull/945#pullrequestreview-1727952706

**Review by Andrew565 — APPROVED** [review]

**Comment by it-harrison** [comment]
> @it-harrison Is it possible to clean up any lone-variables hanging around individual components like this now?
> 
> https://github.com/department-of-veterans-affairs/component-library/blob/5aefd6dcbff87864fae2ef21af98a18cc6acef90/packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss#L4

@jamigibbs we could move these variables into the css-library package? I'm not sure how else to leverage the fact that we are now installing css-library as a dependency.

**Comment by jamigibbs** [comment]
> > @it-harrison Is it possible to clean up any lone-variables hanging around individual components like this now?
> > https://github.com/department-of-veterans-affairs/component-library/blob/5aefd6dcbff87864fae2ef21af98a18cc6acef90/packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss#L4
> 
> @jamigibbs we could move these variables into the css-library package? I'm not sure how else to leverage the fact that we are now installing css-library as a dependency.

That's probably the best approach. Can you write up a ticket for us to follow-up on doing that?  https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new?assignees=&labels=platform-design-system-team&projects=&template=DST-basic_issue_template.md&title=

**Comment by jamigibbs** [comment]
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2266

**Pull request merged**